packages/dx-demo-shell/src/utils.js

```diff
@@ -0,0 +1,6 @@
+export const findOccurrence = (source, strings) => {
```

Comment:
`(source, strings) => strings.some(str => source.includes(str))`

And do we need additional file because of single helper function?